packages/cdk/lib/env/context-app-parameters.ts

```diff
@@ -212,4 +218,28 @@ export class ContextAppParameters {
     }
     return defFilesystem;
   }
+
+  public getEngineType(): string {
+    let engineType: string;
+    switch (this.engineName) {
```

Comment:
Should you ignore case before checking equivalency?

Reply:
Here I was following the pattern of `getDefaultFilesystem()` above, which does not ignore case. Do you suggest I change both to ignore case?

Reply:
Unlike `filesystem`, the engine name is determined from the customer's `agc-project.yml` file. Its good to have some fool proof. If customer uses case in their project files, we should still be able to determine and tag the resources with the right engine metadata.

Reply:
Good point, will update.